### PR TITLE
Some refactoring

### PR DESCRIPTION
**Pull request by xepozz**
Adjusted changes from https://github.com/yiisoft/yii-debug/pull/133

### CODE REVIEW

File: resources/views/index.html [hunk 1]
```diff
@@ -34,46 +34,46 @@
...
             </a>
         </div>
         <div class="yii-debug-toolbar__block">
             <a href="#" title="">Status
-                <span class="yii-debug-toolbar__label yii-debug-toolbar__label_success" data-key="responseStatusCode">200</span>
+                <span class="yii-debug-toolbar__label yii-debug-toolbar__label_success" data-key="response.statusCode">0</span>
             </a>
             <a href="#" title="">Time
-                <span class="yii-debug-toolbar__label yii-debug-toolbar__label_info" data-key="time">0</span>
+                <span class="yii-debug-toolbar__label yii-debug-toolbar__label_info" data-key="startTime">0</span>
```
Comment (samdark) [inline resources/views/index.html:45]: time.start?

File: resources/views/info/summary.html [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-        <span class="yii-debug-toolbar__label" data-key="phpVersion">0</span>
+        <span class="yii-debug-toolbar__label" data-key="php.version">0</span>
     </a>
 </div>
 <script>
     (function () {
         setTimeout(() => {
-            document.querySelector('[data-key=phpVersion]').textContent = YiiDebug.currentSession.phpVersion || 'n/a';
-            document.querySelector('[data-key=phpVersion]').parentElement.href = YiiDebug.viewerUrl + '/info';
+            document.querySelector('[data-key="php.version"]').textContent = YiiDebug.currentSession.web.php.version || 'n/a';
```
Comment (samdark) [inline resources/views/info/summary.html:9]: php.version or web.php.version?

**Review by samdark — APPROVED** [review]
Looks better overall. I like it 👍

File: resources/views/info/summary.html (hunk 2 above)
Comment (xepozz) [reply resources/views/info/summary.html:9]: `php.version` is common for `web.php.version` or `console.php.version` and etc. I'll add `|| console.php.version` a bit later

File: resources/views/index.html (hunk 1 above)
Comment (xepozz) [reply resources/views/index.html:45]: the same as below

**Review by rustamwin — APPROVED** [review]

**Pull request merged**